update.go

```diff
@@ -57,6 +68,30 @@ other options are ignored.
 			Name:  "blkio-weight",
 			Usage: "Specifies per cgroup weight, range is from 10 to 1000",
 		},
+		cli.IntFlag{
+			Name:  "blkio-leaf-weight",
+			Usage: "Specifies tasks' weight in the given cgroup while competing with the cgroup's child cgroups, range is from 10 to 1000, cfq scheduler only",
```

Comment:
IMO we should be specifying "ranges" as part of the help docs. Shouldn't these be all listed inside the runtime-spec?